rim/templates/rim/home.html

```diff
@@ -2,33 +2,50 @@
 {% load i18n %}
 {% load static %}
 
+{% block css %}
+    {{ block.super }}
+    <link rel='stylesheet' type='text/css' href='{% static "rim/css/home.css" %}'>
+    <link rel='stylesheet' type='text/css' href='{% static "rim/css/search.css" %}'>
+{% endblock %}
+
 {% block content %}
+    {% include 'rim/includes/search.html' %}
     <table>
         <thead>
             <tr>
                 <th>
-                    {% trans 'Serial No' %}
+                    {% include 'rim/includes/header_link.html' with ordering='latest_checkout__timestamp' title='Date Checked Out' %}
+                </th>
+                <th>
+                    {% include 'rim/includes/header_link.html' with ordering='serial_no' title='Serial No.' %}
                 </th>
                 <th>
-                    {% trans 'Equipment Type' %}
+                    {% include 'rim/includes/header_link.html' with ordering='equipment_type__type_name' title='Equipment Type' %}
                 </th>
                 <th>
-                    {% trans 'Manufacturer' %}
+                    {% include 'rim/includes/header_link.html' with ordering='manufacturer' title='Manufacturer' %}
                 </th>
                 <th>
-                    {% trans 'Model' %}
+                    {% include 'rim/includes/header_link.html' with ordering='equipment_model' title='Model' %}
                 </th>
                 <th>
-                    {% trans 'Location' %}
+                    {% include 'rim/includes/header_link.html' with ordering='latest_checkout__location' title='Location' %}
                 </th>
                 <th>
-                    {% trans 'Count' %}
+                    {% include 'rim/includes/header_link.html' with ordering='count' title='Count' %}
                 </th>
             </tr>
         </thead>
         <tbody>
             {% for e in equipment_list %}
                 <tr>
+                    <td>
+                        {% if e.latest_checkout == None %}
+                            {% trans 'At ResNet' %}
```

Comment:
You should probably just leave this column blank (`&nbsp;`) if there's no latest_checkout

Reply:
Also, maybe move this column to right before/after the Checkout Location?